cmd/cluster-capacity/app/options/options.go

```diff
@@ -71,7 +72,7 @@ func NewClusterCapacityOptions() *ClusterCapacityOptions {
 
 func (s *ClusterCapacityOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.Kubeconfig, "kubeconfig", s.Kubeconfig, "Path to the kubeconfig file to use for the analysis.")
-	fs.StringVar(&s.PodSpecFile, "podspec", s.PodSpecFile, "Path to JSON or YAML file containing pod definition.")
+	fs.StringVar(&s.PodSpecFile, "podspec", s.PodSpecFile, "Path to JSON or YAML file containing pods definition.")
```

Comment:
It would be better to clarify, it could be a path to single JSON or YAML file if only one spec file is specified, and a dir that contains multiple individual spec files in case of multiple pod spec files are provided. In general, It'd be good too to have a single JSON and YAML file that contains multiple pod definitions.